drivers/gpio/gpio_nrfx.c

```diff
@@ -171,7 +191,18 @@ static int gpio_nrfx_pin_configure(const struct device *port, gpio_pin_t pin,
 		__ASSERT_NO_MSG(err == NRFX_SUCCESS);
 	}
 
-	return 0;
+end:
+#ifdef CONFIG_SOC_NRF54H20_GPD
+	if (cfg->pad_pd == NRF_GPD_FAST_ACTIVE1) {
+		cfg->port->RETAINSET = BIT(pin);
```

Comment:
Please make sure to use HAL (add new function if needed).

Reply:
done